### PR TITLE
Simplification of #ifdef statements

### PR DESCRIPTION
**Pull request by MathieuDutSik**
# Pull Request Summary
 
The issue was raised during the last developer meeting. The PR simplifies the #ifdef statements and
so is a step in the right direction. If This PR passes then other corrections are possible for other files.

## Description

Only removal of CPP processors statements.

### Issue(s) addressed

No issue was raised in the GitHub issues list but there was a discussion about it.

### Check list  

- [ ] Branch is up to date with the authoritative repository (NOAA-EMC) develop branch. 
- [ ] Checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop). 
- [ ] If a version number update is required, checked the [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) checklist. 
- [ ] If a new feature was added, a regression test for testing the new feature is added. 
 
### Testing

* How were these changes tested?
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
* Have the matrix regression tests been run (if yes, please note HPC and compiler)?
* Please indicate the expected changes in the regression test output, (Note the [list](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-develop#4-look-at-results) of known non-identical tests.)
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
@MathieuDutSik  There are duplicated ifdef blocks [here](https://github.com/MathieuDutSik/WW3/blob/95a5a153b38409dcce2286c2e5ea601c58e264cf/model/src/w3wavemd.F90#L1629). Could you remove the duplicates?

**Comment by MathieuDutSik** [comment]
> @MathieuDutSik There are duplicated ifdef blocks [here](https://github.com/MathieuDutSik/WW3/blob/95a5a153b38409dcce2286c2e5ea601c58e264cf/model/src/w3wavemd.F90#L1629). Could you remove the duplicates?

I certainly could but I did not address this file. I propose to consider this block of files (already 11) first and then if all passes correctly, to consider the next block until everything is cleaned.

Is that reasonable to you?

**Comment by MatthewMasarik-NOAA** [comment]
Hi @MathieuDutSik,  thanks for your continued work in this cleanup effort.  I'll start doing my own check on this, then when @JessicaMeixner-NOAA returns tomorrow (Wed 12th) we will both discuss the review.

File: model/src/w3gdatmd.F90 [hunk 1]
```diff
@@ -2299,12 +2271,10 @@ SUBROUTINE W3SETG ( IMOD, NDSE, NDST )
     US3DF  => GRIDS(IMOD)%US3DF
     USSPF  => GRIDS(IMOD)%USSPF
     USSP_WN => GRIDS(IMOD)%USSP_WN
+    FFACBERG => GRIDS(IMOD)%FFACBERG
```
Comment (JessicaMeixner-NOAA) [inline model/src/w3gdatmd.F90:2274]: @MathieuDutSik this line looks added?  Is this intentional? Is this moving something from somewhere else (couldn't find where that was if so, but might have just missed it).

**Comment by JessicaMeixner-NOAA** [comment]
@MathieuDutSik I think it's fine to do this clean-up in batches.  Thanks for this round of clean-up!

File: model/src/w3gdatmd.F90 (hunk 1 above)
Comment (MathieuDutSik) [reply model/src/w3gdatmd.F90:2274]: This was moved from the line below at line 2306. So now new code, just changes to ensure that more W3_REF1 code is put together.

Comment (MatthewMasarik-NOAA) [reply model/src/w3gdatmd.F90:2274]: @MathieuDutSik , okay that checks out.  Thanks for the clarification.

**Review by MatthewMasarik-NOAA — APPROVED** [review]
Code changes look good.  Matrix tests passed with only known non-identicals as changes.
```bash
**********************************************************************           
********************* non-identical cases ****************************           
**********************************************************************           
mww3_test_03/./work_PR2_UNO_MPI_d2                     (15 files differ)         
mww3_test_03/./work_PR1_MPI_d2                     (17 files differ)             
mww3_test_03/./work_PR3_UNO_MPI_d2_c                     (12 files differ)       
mww3_test_03/./work_PR3_UQ_MPI_d2_c                     (15 files differ)        
mww3_test_03/./work_PR3_UNO_MPI_d2                     (12 files differ)         
mww3_test_03/./work_PR2_UQ_MPI_d2                     (15 files differ)          
mww3_test_03/./work_PR3_UQ_MPI_d2                     (15 files differ)          
ww3_ta1/./work_UPD0F_U                     (0 files differ)                      
ww3_tp2.10/./work_MPI_OMPH                     (7 files differ)                  
ww3_tp2.16/./work_MPI_OMPH                     (4 files differ)                  
ww3_tp2.6/./work_ST0                     (1 files differ)                        
ww3_tp2.6/./work_ST4                     (1 files differ)                        
ww3_ufs1.3/./work_a                     (1 files differ)                                                                                                       
**********************************************************************           
************************ identical cases *****************************           
**********************************************************************
```
* [matrixCompSummary.txt](https://github.com/NOAA-EMC/WW3/files/9778862/matrixCompSummary.txt)
* [matrixCompFull.txt](https://github.com/NOAA-EMC/WW3/files/9778863/matrixCompFull.txt)
* [matrixDiff.txt](https://github.com/NOAA-EMC/WW3/files/9778861/matrixDiff.txt)

**Comment by MatthewMasarik-NOAA** [comment]
Thanks @MathieuDutSik !

**Pull request merged**